src/elements/video/demo/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html ng-app="demo">
+<head lang="en">
+  <meta charset="UTF-8">
+  <title></title>
+  <link rel="stylesheet" href="../../../../node_modules/semantic-ui-css/semantic.css"/>
+  <style>
+    body {
+      padding: 2em;
+    }
+  </style>
+</head>
+<body ng-controller="VideoDemoCtrl">
+
+  <div class="ui stackable centered grid">
+    <div class="row">
+      <div class="six wide column">
+        <h1 class="ui header">Youtube</h1>
+        <sm-video source="youtube" data-id="O6ROCaLvMnQ" placeholder="http://placekitten.com/601/300" pr-theme="light" options="options"
+                  on-ready="onReady(player)" pr-enablejsapi="1"></sm-video>
+      </div>
+    </div>
+    <div class="row">
+      <div class="six wide column">
```

Comment:
In the demo, the controls are disabled until the player is ready. 
When a directive is used with placeholder option, the iframe is lazy-loaded after the placeholder trigger, either by clicking or pressing space key. That's how semantic implements its placeholder. It'd be good for page load performance (iframes are heavy) but it prevents the use of player API before the click. Should we have this behavior as option instead? @m0t0r @caitp 

With that said, in the demo, it's weird that the vimeo controls are disabled too since it doesn't use placeholder. @m0t0r Can you check the log?

For the second note, showing controls is to demonstrate the use of player API. It's a part of the demo, not the directive.